training/backends/kubernetes.js

```diff
@@ -0,0 +1,201 @@
+// -*- mode: js; indent-tabs-mode: nil; js-basic-offset: 4 -*-
+//
+// This file is part of Almond
+//
+// Copyright 2019 The Board of Trustees of the Leland Stanford Junior University
+//
+// Author: Giovanni Campagna <gcampagn@cs.stanford.edu>
+//
+// See COPYING for details
+"use strict";
+
+const k8s = require('@kubernetes/client-node');
+const Tp = require('thingpedia');
+
+const Config = require('../config');
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
```

Comment:
how do you plan to setup the default kube credentials?

Reply:
It should load from cluster credentials (service account)

Reply:
I'm not sure if the default service account inside the pod has permission to create job. You can give it a try.

Reply:
No I mean we need to add the ServiceAccount declarations to the training daemon Pod. I haven't done that yet.